samples/go/photo-dedup/model/photogroup.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package model
+
+import (
+	"github.com/attic-labs/noms/go/types"
+	"github.com/attic-labs/noms/samples/go/photo-dedup/dhash"
+	"github.com/attic-labs/noms/go/d"
+)
+
+type PhotoGroup interface {
```

Comment:
I think we want to avoid these kind of wrapper datatypes. If we need to improve something in marshaling or elsewhere, let's do that, but we don't want to have to create these.